### PR TITLE
change language dropdown to use the local names

### LINKED ISSUE #516: Use only the languages own language translation
Currently we translate every language name to every other language which results in having german translated to "Deutsch, German, Allemand...". This doesn't help anyone to find the right language, as every german will recognize the "Deutsch" language string. This is also the behaviour the android version has.

### PR DESCRIPTION
**Pull request by Simon-Laux**
closes #516

### CODE REVIEW

File: bin/extract_local_language_names.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const { readdirSync, readFileSync, writeFileSync } = require('fs')
+const { join } = require('path')
+
+const localeDir = join(__dirname, '..', '_locales')
+
+// English Lang as fallback
+const englishLang = JSON.parse(readFileSync(join(localeDir, 'en.json')))
```
Comment (ralphtheninja) [inline bin/extract_local_language_names.js]: Note that you can just as well do

```js
const englishLang = require(join(localeDir, 'en.json'))
```

File: bin/extract_local_language_names.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+function getAvailableLanguages () {
+  return readdirSync(localeDir)
+    .filter(l => {
+      return !l.startsWith('_') && l.endsWith('.json')
+    })
+    .map(l => {
+      const locale = l.split('.json')[0]
+      const content = JSON.parse(readFileSync(join(localeDir, l)))
```
Comment (ralphtheninja) [inline bin/extract_local_language_names.js]: And here.

File: bin/extract_local_language_names.js (hunk 1 above)
Comment (Jikstra) [reply bin/extract_local_language_names.js]: We should prefer to use require as webpack has good support for it. Sure on a require you could somehow inline js code which gets executed but this shouldn't be a problem.

Comment (Simon-Laux) [reply bin/extract_local_language_names.js]: I didn't thought of that, thanks for pointing it out to me.

**Comment by ralphtheninja** [comment]
@Simon-Laux I can solve the conflict if you like.

**Comment by Simon-Laux** [comment]
> @Simon-Laux I can solve the conflict if you like.

Go ahead, thanks.

**Comment by Simon-Laux** [comment]
I confirm that it works for me

**Pull request merged**